pkg/kubelet/cm/helpers_linux_test.go

```diff
@@ -197,3 +200,39 @@ func TestMilliCPUToQuota(t *testing.T) {
 		}
 	}
 }
+
+func TestHugePageLimits(t *testing.T) {
```

Comment:
Can you make this a table driven test and use subtests to separate cases?
Table-driven: https://github.com/golang/go/wiki/TableDrivenTests
Subtests: https://blog.golang.org/subtests